### ISSUE TITLE
Add Account Settings Page

### ISSUE THREAD

**Issue by anandology** [AamAadmiParty/cleansweep#183]
Add account settings page, showing name, email, phone number. Eventually we need the ability to edit all of them, but email and phone number requires validation before updating.  So, just add the ability to edit the name and show phone number and email as read-only.

Open Issues:

- Would it be better to have that profile page, with option to edit the profile? That way we can even let the admins edit profiles of users.

**Comment by charanjeetaap** [comment]
Yes we need the profile page. And right now we are not verifying phone
number entered by a volunteer. We need to do that too.

**Comment by anandology** [comment]
I think instead of adding a settings page, we can just have a profile page with an option to edit the profile. 

The profile page should allow changing the logged in user to change name and phone number, and admin to change name phone number and email.

**Comment by charanjeetaap** [comment]
Also Profile Pic, if possible.

**Comment by anandology** [comment]
No, we don't have infrastructure to store phones. We use photo from google / fb profile.
